src/ScrollableContainers/_tk.py

```diff
@@ -39,13 +41,41 @@ def __init__(self, *args, **kwargs):
         self._window = self._canvas.create_window((0, 0), window=self.frame, anchor=tk.NW)
         self.frame.bind("<Configure>", self._on_frame_configure)
         self._on_frame_expose_id = self.frame.bind("<Expose>", self._on_frame_expose)
+        self._hide_scrollbars_id = None
 
         # Initially, the vertical scrollbar is a hair below its topmost
         # position. Move it to said position. No harm in doing the equivalent
         # for the horizontal scrollbar.
         self._canvas.xview_moveto(0.0)
         self._canvas.yview_moveto(0.0)
 
+    def _show_scrollbars(self):
+        """
+        Move the horizontal and vertical scrollbars above the scrollable
+        canvas.
+        """
+        self._xscrollbar.lift()
+        self._yscrollbar.lift()
+
+    def _hide_scrollbars(self):
+        """
+        Move the horizontal and vertical scrollbars below the scrollable
+        canvas.
+        """
+        self._xscrollbar.lower()
+        self._yscrollbar.lower()
+
+    def _peek_scrollbars(self, ms: int = 1000):
+        """
+        Show the horizontal and vertical scrollbars. Hide them after a delay.
+
+        :param ms: Delay in milliseconds.
+        """
+        if self._hide_scrollbars_id:
+            self.after_cancel(self._hide_scrollbars_id)
+        self._show_scrollbars()
+        self._hide_scrollbars_id = self.after(ms, self._hide_scrollbars)
```

Comment:
If the cursor is on a scrollbar, it should not be hidden until it moves out.

Reply:
Fixed in f386a718e817dad6d08737cfbd59b62dcce6eb18. However, if the mouse button is held down on the scrollbar, then moved out of it, the scrollbars are hidden. This should not happen, either.

Reply:
Tried this in e176069e61ef9c51f18ed352f66f56666b2d15bb, but did not work as expected. Reverted it.